package.json

```diff
@@ -11,7 +11,8 @@
     "lint": "eslint . --ext .js,.ts",
     "lint:fix": "eslint . --ext .js,.ts --fix",
     "test": "mocha --exit --recursive 'test/**/*.test.ts'",
-    "build": "tsc"
+    "build": "tsc",
+    "prepare": "npm run build"
```

Comment:
Why `npm` when we are using `yarn` for this project?

Reply:
If we change the script to `"prepare": "yarn run build"` it will fail if we execute it by `npm run prepare`.
The current one works with both `npm run prepare` and `yarn run prepare`.